tests/contrib/django_util/test_decorators.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+
+from django import http
+import django.conf
+from django.contrib.auth.models import AnonymousUser, User
+import mock
+from six.moves import http_client
+from six.moves import reload_module
+from six.moves.urllib import parse
+from tests.contrib.django_util import TestWithDjangoEnvironment
+
+import oauth2client.contrib.django_util
+from oauth2client.contrib.django_util import decorators
+
+
+class OAuth2EnabledDecoratorTest(TestWithDjangoEnvironment):
+
+    def setUp(self):
+        super(OAuth2EnabledDecoratorTest, self).setUp()
+        self.save_settings = copy.deepcopy(django.conf.settings)
+
+        if hasattr(django.conf.settings, 'GOOGLE_OAUTH2_STORAGE_MODEL'):
+            del django.conf.settings.GOOGLE_OAUTH2_STORAGE_MODEL
+
+        # OAuth2 Settings gets configured based on Django settings
+        # at import time, so in order for us to reload the settings
+        # we need to reload the module
+        reload_module(oauth2client.contrib.django_util)
+        self.user = User.objects.create_user(
+            username='bill', email='bill@example.com', password='hunter2')
+
+    def tearDown(self):
+        super(OAuth2EnabledDecoratorTest, self).tearDown()
+        django.conf.settings = copy.deepcopy(self.save_settings)
+
+    def test_no_credentials_without_credentials(self):
+        request = self.factory.get('/test')
+        request.session = self.session
+
+        @decorators.oauth_enabled
+        def test_view(request):
+            return http.HttpResponse("test")  # pragma: NO COVER
+
+        response = test_view(request)
+        self.assertEquals(response.status_code, http_client.OK)
```

Comment:
Yes sorry if there was misunderstanding but what I was trying to say is that I believe it is assertEqual because it really just matters that the response status code is 200.